src/main/database/init/init_schema.sql

```diff
@@ -0,0 +1,66 @@
+create database FUNDY;
+use FUNDY;
+
+# Objects
+create table account(
+    uuid int auto_increment,
+    budget int,
+
+    primary key(uuid)
+);
+
+create table user (
+    uuid int auto_increment,

```

Comment:
uuid를 여러 테이블에서 사용한다면 이름을 명시하는 것이 좋을 것 같습니다.
(다른 테이블도 uuid보다 다른 이름이 좋을 것 같아요)
```suggestion
    user_uuid int auto_increment,
```